index.html

```diff
@@ -5,39 +5,31 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Book List</title>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-F3w7mX95PdgyTmZZMECAngseQB83DfGTowi0iMjiWaeVhAn4FJkqJByhZMI3AhiU" crossorigin="anonymous">
 </head>
 <body>
   <div class="container">
     <div id="errors"></div>
-    <h1>Awesome Books</h1>
-    <form id="book-form">
-      <div>
-        <label for="title">Title</label>
-        <input type="text" id="title" class="u-full-width" >
+    <h1 class="text-center text-dark mt-2">Awesome Books</h1>
+
+    <div class="border border-dark p-3 w-100 mb-4" id="book-list">
+     
+    </div>
+    <form class="d-flex align-items-center flex-column w-100" id="book-form">
+      <h3 class="text-center text-dark">Add a new book</h3>
+      <div class="w-50">
+        <input type="text" id="title" placeholder="Title" class="mb-3 p-1 w-100 border-dark text-dark" >
       </div>
-      <div>
-        <label for="author">Author</label>
-        <input type="text" id="author" class="u-full-width" >
+      <div class="w-50">
+        <input type="text" id="author" placeholder="Author" class="mb-3 p-1 w-100 border-dark text-dark" >
       </div>
-      <div>
-        <label for="isbn">ISBN#</label>
-        <input type="text" id="isbn" class="u-full-width" required>
+      <div class="w-50">
+        <input type="text" id="isbn" class="mb-3 p-1 w-100 border-dark text-dark" placeholder="ISBN" required>
       </div>
-      <div>
-        <input type="submit" value="Add" class="u-full-width">
+      <div class="d-flex align-items-center w-50 justify-content-end">
+        <input type="submit" value="Add" class="border-dark text-dark mb-3">
       </div>
     </form>
```

Comment:
- [ ] Please add a `navigation` bar that has 3links: booklist, add book form, and contact info
- [ ] And you need to add a date below the navigation bar and you should use [Luxon](https://moment.github.io/luxon/#/) to get the current date and time
- [ ] Remember to add a `footer` as well

Reply:
Check out the design image for reference
![nsv](https://user-images.githubusercontent.com/44745855/134316542-71b32d45-c14d-46e9-9b33-e66da8be0894.png)

Reply:
I accidentally submitted day3 project for day 4